### PR TITLE
Add verbosity flags

### LINKED ISSUE #127: Toggle ansible verbosity via rho scan command flag & log scan output to file
ansible-playbook has several verbosity flags -v, -vv -vvv -vvvv. We can expose this verbosity via the rho scan command line with similar flags to improve debug ability.

To improve SEAP debugging the output of the ansible-playbook command should be written to the file system by default, we can supply an optional flag to specify the output log location.

### PR DESCRIPTION
**Pull request by noahl**
Add a verbosity flag that works the same way that Ansible's does (-v,
-vv, -vvv, and -vvvv for different levels of verbosity). Propagate it
to Ansible in 'rho scan'.

Also use the verbosity to set the Python logging level, so that Rho
itself can use the native Python logging.

Closes #127.

### CODE REVIEW

**Comment by chambridge** [comment]
Code looks good, but let's add some documentation to the [command syntax usage doc](https://github.com/quipucords/rho/blob/master/doc/command_syntax_usage.rst#command-syntax--usagel).

There you can describe the verbosity flag and the output log file flag form part 1.

**Comment by noahl** [comment]
@chambridge I added documentation. Please take a look.

File: doc/command_syntax_usage.rst [hunk 1]
```diff
@@ -68,6 +68,19 @@ to pass in auths as per the priority they deem fit for a profile. If a non root
...
 
+The output of the Ansible process is saved to `data/scan_log` by
+default, for debugging. This location can be changed with the
+`--logfile` flag.
+
+^^^^^^^^^^^^
+Common Flags
+^^^^^^^^^^^^
+
+All Rho commands accept the `-v` flag, which increases the verbosity
```
Comment (chambridge) [inline doc/command_syntax_usage.rst]: Lowercase when we refer to rho in the documentation.
"All rho commands accept the `-v` flag, which increases the verbosity of the rho command line output."

**Review by chambridge — APPROVED** [review]
One minor comment on the documentation. Otherwise LGTM!

**Comment by noahl** [comment]
@chambridge fixed, thanks!

**Pull request merged**